data/free worlds start.txt

```diff
@@ -1399,6 +1399,162 @@ mission "FW Pirates 3.1B"
+		system Shaula
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		ship "Bastion (Heavy)" "Dread"
+		ship "Carrier (Alpha)" "Trafficker"
+		ship "Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
```

Comment:
Thanks for the feedback. I inflated the enemy fleet because of some of Kiko's remarks that none of the fights in the game reflect the true scale of what a planetary conquest would look like, and I was trying to make this challenging for someone who has built up a large and powerful fleet prior to starting the storyline. I'll revise the numbers down, and maybe replace some of the small stuff with larger, more dangerous foes.

To answer your original question about the Alpha carrier, I meant that to be a reference to the fact that the Alphas depend on having many fertile young slaves to spawn. Greenrock being the only major slave market in known space, it seemed like an interesting twist to throw in. Also, it would make the fight significantly tougher.